README.md

```diff
@@ -105,3 +106,30 @@ jobs:
 and as soon as the job is finished you will see the test results in the pull request status:
 
 ✅ | ❌ Testing Farm - CentOS 7 - Build finished
+
+### Merge request example
```

Comment:
I would rename this, because there is no really a difference between pull request (above) and merge request.
Therefore, from user's point of view, I would not know the difference between these two examples.

Reply:
Can be. Or `Run workflow at push to the main branch.`